modules/glossary-of-common-terms-for-rhoai.adoc

```diff
@@ -0,0 +1,85 @@
+//Reference:: IBM glossary https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x9721361
+
+cluster storage:: A persistent volume that retains the files and data within a workbench. A workbench has access to one or more cluster storage instances.
+
+custom resource (CR):: A resource implemented through the Kubernetes CustomResourceDefinition API. A custom resource is distinct from the built-in Kubernetes resources, such as the pod and service resources. Every CR is part of an API group.
+//Reference: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#custom-resource
+
+custom resource definition (CRD):: In Red Hat OpenShift, a custom resource definition (CRD) defines a new, unique object `Kind` in the cluster and lets the Kubernetes API server handle its entire lifecycle.
+// Reference:: https://redhat-documentation.github.io/supplementary-style-guide/#custom-resource-definition
+
+data connection:: The configuration parameters required to connect to a data source, such as an S3 object bucket.
+
+data science pipelines:: A portable machine learning (ML) workflow to develop and deploy data science models.
+
+data science project:: An isolated environment for developing, training and serving machine learning models.
+
+disconnected environment:: An environment on a restricted network that does not have an active connection to the Internet.
+// Reference: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#disconnected
+
+distributed workloads:: Data science workloads that are run simultaneously across multiple nodes in an OpenShift cluster.
+
+fine-tuning:: The process of adapting a pre-trained model to perform a specific task by conducting additional training. Fine tuning may involve (1) updating the model’s existing parameters, known as full fine tuning, or (2) updating a subset of the model’s existing parameters or adding new parameters to the model and training them while freezing the model’s existing parameters, known as parameter-efficient fine tuning.
+// Reference: RHEL AI glossary, https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x9094307
+
+graphics processing unit (GPU):: A specialized processor designed to rapidly manipulate and alter memory to accelerate the creation of images in a frame buffer intended for output to a display. GPUs are heavily utilized in machine learning due to their parallel processing capabilities.
+//Reference IBM Glossary https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x8987320
+
+inference:: The process of using a trained AI model to generate predictions or conclusions based on the input data provided to the model. 
+
+inference server:: A server that helps an AI model make new conclusions based on its prior training. Inference servers feed the input requests through a machine learning model and return an output.
```

Comment:
> A server that helps an AI model make new conclusions based on its prior training.

What about "server that performs inference"? One difficulty here may be that "server" probably means a process (executing software program), not a hardware machine.